### PR TITLE
python.yaml updates for openSUSE (1 of 11)

### PR DESCRIPTION
**Pull request by neotinker**
I have a lot of updates for openSUSE and was asked to break them up for easier digestion per Draft PR #28385 

These keys are not necessarily related to each other. I'm just processing these commits alphabetically and breaking them up into groups of approx. 10.

https://software.opensuse.org/package/python-ipython
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-ipython/standard
https://software.opensuse.org/package/python2-argcomplete
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-argcomplete/standard
https://software.opensuse.org/package/python
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.15905/standard
https://software.opensuse.org/package/python3-avahi
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.13855/standard
https://software.opensuse.org/package/python2-cairo
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-cairo/standard
https://software.opensuse.org/package/python3-CairoSVG
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-CairoSVG/standard
https://pypi.org/project/chainer/
https://software.opensuse.org/package/python-Cheetah
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-Cheetah/standard
https://software.opensuse.org/package/python2-coverage
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-coverage/standard
https://software.opensuse.org/package/python2-pycrypto
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python-pycrypto/standard

### CODE REVIEW

**Review by nuclearsandwich — APPROVED** [review]

File: rosdep/python.yaml [hunk 1]
```diff
@@ -204,6 +204,7 @@ ipython:
   gentoo: [dev-python/ipython]
   macports: [py27-ipython]
   nixos: [pythonPackages.ipython]
+  opensuse: [python2-ipython]
```
Comment (mabelzhang) [inline rosdep/python.yaml:207]: I think the links provided are for `python-ipython` (and `python3-ipython`), as opposed to `python2-ipython`. Adding links for the latter for future reference
https://software.opensuse.org/package/python2-ipython
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/python2-ipython/standard

Could you confirm the key currently in the file is the one you want?

**Review by mabelzhang — APPROVED** [review]
Small note to double check, otherwise can merge.

File: rosdep/python.yaml (hunk 1 above)
Comment (neotinker) [reply rosdep/python.yaml:207]: Yes, that was my mistake. The key is correct. I provided the wrong links.

Comment (mabelzhang) [reply rosdep/python.yaml:207]: Thanks!

**Pull request merged**